Assets/scss/components/article-cards.scss

```diff
@@ -0,0 +1,95 @@
+.article-cards{
+    background-color: $white-smoke;
+    font-family: $arial;
+    max-width: 100%;
+    margin: 0;
+
+    &__menu{
+        display: flex;
+        padding: 20px;
+       
+        &--item{
+            @include article-dropdown-style;
+            max-width: 150px;
+        }
+        .dopdown-up{
```

Comment:
you have a typo here. I know those are annoying, but they often cause weird bugs